### ISSUE TITLE
[iOS] Wrong value of ExposureInfo's Duration

### ISSUE THREAD

**Issue by pavshr** [xamarin/ExposureNotification.Sample#83]
Hello, I do observe some strange values for **Duration** property in **ExposureInfo** object on iOS devices in **0.11.0-preview**, e.g., `{1.06:00:00}, {1.01:00:00}`. The phones where exposed to each other for about 20-25 minutes.
Meanwhile, on Android it looks more like a real time intervals, e.g. `00:25:00`. Is it a bug, or am I missing something here?

Here is a screenshot from debugger in VS:
<img width="621" alt="image" src="https://user-images.githubusercontent.com/14860255/85305298-287f6180-b4ad-11ea-8b12-b19b3b9e64d8.png">

Thank you in advance :)

**Comment by Redth** [comment]
@pavshr we are following [Apple's docs](https://developer.apple.com/documentation/exposurenotification/enexposureinfo/3583714-duration) which state the `ENExposureInfo.duration` is in minutes (we parse it here: https://github.com/xamarin/XamarinComponents/blob/master/XPlat/ExposureNotification/source/Xamarin.ExposureNotification/ExposureNotification.ios.cs#L278 ).   It seems like at runtime maybe Apple's API is actually in seconds?

**Comment by pavshr** [comment]
@Redth 
Yes, I agree. There must be something that does not match in Apple docs, because if we have look at the type of the `duration` instance property of `ENExposureInfo` [here](https://developer.apple.com/documentation/exposurenotification/enexposureinfo/3583714-duration), it is **TimeInterval**, which is according to [the docs](https://developer.apple.com/documentation/foundation/timeinterval) is a number of seconds. The description, however, says that `duration` is 
> The length of time in minutes that the contact was in proximity to the user.

**Comment by Redth** [comment]
@pavshr so I think it’s safe to assume we can change this to expect seconds. We will make this update.
